detectable/src/main/java/com/blackduck/integration/detectable/detectables/opam/parse/OpamFileParser.java

```diff
@@ -70,12 +70,24 @@ public Map<String, String> parseData(List<String> lines) {
                 continue;
             }
 
-            if (line.startsWith("version:")) { // parse version
-                output.put(VERSION, line.split(":")[1]);
+            if (line.startsWith("version:")) {
+                //parse version
+                String version = line.split(":")[1];
+                Matcher matcher = pattern.matcher(version);
+                if (matcher.find()) {
+                    version = matcher.group(1);
+                }
+                output.put(VERSION, version);
             }
 
-            if (line.startsWith("name:")) { // parse package name
-                output.put(NAME, line.split(":")[1]);
+            if (line.startsWith("name:")) {
```

Comment:
There's an opportunity for code deduplication between parsing out name and version values.
You could define a method that's something like `putValueIfLineSpecifiesIt(String key, Map<String, String> output)`.